reflect.go

```diff
@@ -101,6 +101,9 @@ func applyTags(f *field, tags map[string]string) {
 		// Probably shouldn't be HTML but whatever.
 		f.Footer = template.HTML(v)
 	}
+	if v, ok := tags["class"]; ok {
+		f.Class = template.HTMLEscapeString(v)
```

Comment:
What is the reasoning behind doing HTMLEscapeString? I'm trying to think of a case where this would have something that needs escaped as part of the class and none are coming to mind.